src/pages/Account/Login.tsx

```diff
@@ -137,13 +140,13 @@ export const Login: React.FC<LoginProps> = ({
             displayRegisterForm={displayRegisterForm}
             displayPasswordResetForm={displayPasswordResetForm}
             closeAuthenticationModal={() => null}
-            afterUserIsLoggedIn={redirectAfterLogin}
+            afterUserIsLoggedIn={afterUserIsLoggedIn}
           />
         )}
         {formToDisplay === "passwordReset" && (
           <PasswordResetForm
             displayLoginForm={displayLoginForm}
-            closeAuthenticationModal={redirectAfterLogin}
+            closeAuthenticationModal={afterUserIsLoggedIn}
```

Comment:
should we call this method? it's not a login.. this seems wrong.

Reply:
fixed in b54905562aeb508beb223e2d87710f875297d159